content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,115 @@
+In recent years, digital DJ controllers became more popular, but there are still many DJs that stick with their Technics SL-1210s turntables.
+
+There's a reason for that: the haptic feeling of manipulating music with your hands is still very different from what most controllers can do.
+Motorized controllers like the Rane One try to provide a similar workflow, but it's still to be seen if they will have a notable impact on the prevalence of vinyl.
+
+On the other hand, going digital has its benefits:
+Jumping to different positions inside a track using hotcues just isn't possible with vinyl records, and carrying around heavy crates of records isn't the best you can do for your back.
+
+Digital Vinyl Systems (DVS) aim to provide the best of both worlds and combine the flexibility and power of digital DJing with the haptics of traditional vinyl DJing.
+But how does it actually work?
+
+## The Basics
+
+Instead of a traditional record that contains music, a "control" record is used.
+Such a record contains a special signal that is processed by the computer.
+It transforms the audio data into information about direction ("Is the record playing forwards or backwards?"), pitch ("How fast is the record spinning?") and position information ("At which position of the record is the needle?") and uses that track to manipulate the playing track in software.
+
+There are many formats out there:
+[VirtualDJ](http://www.virtualdj.com/buy/controlvinyl.html), [Rekordbox](https://www.pioneerdj.com/product/features/software/rekordbox-dvs-control-vinyl/), [Numark](https://www.numark.com/product/virtualvinyl), [MixVibes](https://www.mixvibes.com/cross-dj-4/), [Traktor](https://www.native-instruments.com/en/products/traktor/digital-vinyl/traktor-scratch-a10/) and [Serato](https://serato.com/dj/pro/expansions/dvs?dvs=dvs-ready) all provide their own solution.
+Mixxx (and the underlying [xwax](https://xwax.org/) library) [support the latter 3 formats](https://manual.mixxx.org/2.3/en/chapters/vinyl_control.html#supported-timecode-media).
+
+Serato is one of the most popular and robust digital vinyl systems.
+There's a also Serato Control CD which basically works the same, just for CDJ instead of turntables.
+Since the latter essentially works the same as the vinyl version and can be [downloaded for free from the Serato Website](https://serato.com/controlcd/downloads), I'm going to explain how it works based on that format.
+
+Let's take a look at the timecode signal.
+If you open the WAV file with an audio editor like Audacity, it looks like this:
+
+![Serato CD Timecode Signal in Audacity]({static}/images/news/timecode-signal.png)
+
+As you can see, the timecode signal is a stereo signal.
+The left channel is on top and the right channel is on the bottom.
+Both channels look very similar, but the right channel is shifted a bit.
```

Comment:
```suggestion
Both channels look very similar, but the right channel is shifted a little.
```
I was slightly confused by "bit" because of the context of encoding a binary data stream.